addon/utils/pretty-percentage.js

```diff
@@ -0,0 +1,10 @@
+export default function prettyPercentage(decimalValue, precision=0) {
```

Comment:
I'd think this should be a combo helper and util though, namely as this is presentational in nature. Thoughts?